charts/super-agent/charts/super-agent-deployment/templates/deployment-superagent.yaml

```diff
@@ -54,8 +54,13 @@ spec:
           image: {{ include "newrelic.common.images.image" ( dict "imageRoot" .Values.image "context" .) }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
 
-          {{- with .Values.extraEnv }}
           env:
+            - name: NR_LICENSE_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "newrelic.common.license.secretName" . }}
+                  key: {{ include "newrelic.common.license.secretKeyName" . }}
+          {{- with .Values.extraEnv }}
```

Comment:
I would add an extra env var NR_SA_OPAMP__HEADERS__API-KEY so that we van remove the "opamp:" entry in the sa config

Reply:
done